### ISSUE TITLE
Colorbox resize on click of Alt key

### ISSUE THREAD

**Issue by arpitasethi** [jackmoore/colorbox#869]
Colorbox pop-up size gets changed when user clicks on Alt button on FF and IE11 browser(windows), after the popup is opened.
Is it a known bug? Any recommendation?

**Comment by jackmoore** [comment]
Sorry, I could not reproduce this issue. Pressing alt in IE11 did toggle the menu bar, which does affect the viewport size. Colorbox's width can be set as a percentage of viewport size and will resize and reposition when the viewport size changes (window.resize event). Could that be the issue? If not I'll need more information for reproducing the problem.

**Comment by arpitasethi** [comment]
Yes, that is the issue. Thanks for the idea, I have a workaround at least. :)
